src/app/components/LayerPicker/LayerDetails.tsx

```diff
@@ -70,14 +70,14 @@ const getDetails = (t: TFunction): Details => ({
     },
     [Layer.pontusxdev]: {
       description: t('layerPicker.testnet.pontusxdev'),
-      rpcHttp: 'https://pontusx.not.si:443',
+      rpcHttp: 'https://rpc.dev.pontus-x.eu/',
```

Comment:
Seems OK, found these values in Slack messages.

Reply:
I found these in https://docs.pontus-x.eu/docs/Pontus-X%20Testnet/quick_start#setup-metamask